sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -40,6 +41,12 @@ export interface GetTokenOptions {
     abortSignal?: AbortSignalLike;
     claims?: string;
     enableCae?: boolean;
+    proofOfPossessionOptions?: {
```

Comment:
are all of the options truly optional / have defaults?

Reply:
Options don't have a default value. For shr pop, all 3 of them are required.

Reply:
I thought that the flow starts with either `isEnabled: true` or an empty proofOfPossessionOptions object, and only later do the rest of properties get added to the GetTokenOptions bag. 

In that case, I don't think we can make the properties here required as they start out as empty and then get added as part of the flow. @christothes - does that align with your understanding or did I miss something?